API/app.py

```diff
@@ -13,12 +13,12 @@
 from .models import DB
 
 # import model
-from nearest_neighbors_model import predict
+from API.nearest_neighbors_model import predict
 
 def create_app():
     """Create and configure an instance of the Flask application"""
     app = Flask(__name__)
-    app.config['SQLALCHEMY_DATABASE_URI'] = config("DB_URL")
+    app.config['SQLALCHEMY_DATABASE_URI'] = os.getenv("DB_URL")
```

Comment:
Make sure this is reproducible. config was used in class probably for a good reason. It is also what I used last BW.